README.md

```diff
@@ -1,6 +1,15 @@
 ## Butterfree
 _A tool for building feature stores. Transform your raw data into beautiful features._
 
+
+[![Release](https://img.shields.io/github/v/release/quintoandar/butterfree)]((https://pypi.org/project/butterfree/))
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+![Publish](https://github.com/quintoandar/butterfree/workflows/Publish/badge.svg)
+![Test](https://github.com/quintoandar/butterfree/workflows/Test/badge.svg)
+[![Documentation Status](https://readthedocs.org/projects/butterfree/badge/?version=latest)](https://butterfree.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Maybe we could organize this badges in a "CI-related" context table. Maybe already with a placeholder for the future sonar check badge (that is blocked for now).